syntax = "proto3";

package checkuser;

enum RoleUser {
    ADMIN = 0;
    USER = 1;
}

message CheckUserRequest {
    string access_token = 1;
}

message CheckUser {
    int32 status = 1;
    int32 role = 2;
}

service CheckAuthUser {
    rpc GetAuthInfo (CheckUserRequest) returns (CheckUser) {}
}